### PR TITLE
Build the output files as a UMD module

### PR DESCRIPTION
**Pull request by pguimera**
As the title says, I've updated webpack's output to build the file as a [UMD module](https://github.com/umdjs/umd) to improve the library's compatibility when using the built file directly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ npm-debug.log
 _book
 lib
 _site
+.idea
```
Comment (gpbl) [inline .gitignore]: If you don't mind I'd skip this change, as it's not related to the source code :)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-day-picker",
-  "version": "2.4.1",
+  "version": "2.4.2",
```
Comment (gpbl) [inline package.json]: I'd prefer to cut a minor release (hence `2.5.0`)

File: .gitignore (hunk 1 above)
Comment (pguimera) [inline .gitignore]: Fixed 👍

File: package.json (hunk 2 above)
Comment (pguimera) [inline package.json]: Updated

**Comment by pguimera** [comment]
I've cleaned the history a bit whilst applying the suggested changes. Also, in regards to the https://unpkg.com _integration_ there's additional steps to be made (based from this changes) which are specified in the `SUGGESTED WORKFLOW` section of said page.

**Comment by gpbl** [comment]
Thanks a lot! I'll work on the unpkg thing before releasing on npm.

**Comment by gpbl** [comment]
Hey @pguimera,

I've published v2.5.0. It works now from unpkg as well 🙌

**Comment by pguimera** [comment]
Awesome news @gpbl, thanks for that.

I can now use this amazing library in my project without having to do weird tricks in my tooling 😄

**Pull request merged**